tst/EnergyPlus/unit/DaylightingManager.unit.cc

```diff
@@ -855,6 +855,9 @@ TEST_F(EnergyPlusFixture, DaylightingManager_GetDaylParamInGeoTrans_Test)
     ScheduleManager::UpdateScheduleValues(*state);
     InternalHeatGains::GetInternalHeatGainsInput(*state);
     state->dataInternalHeatGains->GetInternalHeatGainsInputFlag = false;
+    for (int SurfNum = 1; SurfNum <= state->dataSurface->TotSurfaces; SurfNum++) {
+        state->dataSurface->SurfWinActiveConstruction(SurfNum) = state->dataSurface->Surface(SurfNum).Construction;
+    }
```

Comment:
Perhaps `state->dataSurface->SurfWinActiveConstruction(SurfNum) = state->dataSurface->Surface(SurfNum).Construction` should be added to whatever function is setting `Surface(SurfNum).Construction` initially. Then it will always be initialized.

Reply:
The assignment is added to the end of `GetSurfaceData` and these initialization are removed.